### ISSUE TITLE
nsave no longer functioning?

### ISSUE THREAD

**Issue by hdugan** [GLEON/GLM-source#56]
A student is trying to output .csv physics files at an hourly timestep. I noticed that the nsave parameter in &output is defunct. Seems like it's hardcoded to be = 24 in glm_init.c

Any workarounds on this? 
Thanks.

**Comment by matthipsey** [comment]
Hi Hilary

The lake.csv file is only available with daily summaries, as it integrates the heat and volume fluxes for daily reporting. This is necessary as inflows and outflows are done daily and can’t be broken down to hourly.

I am pretty sure though that nsave =1 will write the point csv files (the ones at a chosen depth) at the user’s chosen model tilmestep. These do not have the volume and heat flux info like lake.csv, but the user can specify variables at that point such as temperature, light etc. If you want more than this then I think you need to go into the output.nc file directly via R

Cheers
Matt




On 14 Jul 2017, at 11:59 am, hdugan <notifications@github.com<mailto:notifications@github.com>> wrote:


A student is trying to output .csv physics files at an hourly timestep. I noticed that the nsave parameter in &output is defunct. Seems like it's hardcoded to be = 24 in glm_init.c

Any workarounds on this?
Thanks.

**Comment by hdugan** [comment]
Ah, got it. Thanks Matt. 
Is there any work around to getting hourly heat fluxes?

**Comment by jread-usgs** [comment]
@hdugan which fluxes are you looking to get?

**Comment by hdugan** [comment]
@jread-usgs Sensible and latent I think. It's not a deal breaker, but would be helpful.

**Comment by matthipsey** [comment]
are they not in the NC file?


On 17 Jul 2017, at 10:14 pm, hdugan <notifications@github.com<mailto:notifications@github.com>> wrote:


@jread-usgs<https://github.com/jread-usgs> Sensible and latent I think. It's not a deal breaker, but would be helpful.

**Comment by lawinslow** [comment]
No, GLM has a few diagnostic values that end up in the lake.csv and not the NC file.
